lib/diagnostic.md

```diff
@@ -29,7 +33,10 @@ DOM node. (Assume the list item has a class of "listerItem").
 Give two examples of jQuery getter methods and two examples of jQuery setter methods.
 
 <!-- your answer starts here -->
-
+.text() : getter
+.text(text) : setter
+.click() : getter
```

Comment:
.click is questionable as a getter I just checked the docs on this myself

"As the .click() method is just a shorthand for .on( "click", handler ), detaching is possible using .off( "click" )."